roles/marathon/meta/main.yml

```diff
@@ -122,6 +122,8 @@ galaxy_info:
   #- web
 dependencies:
   - role: consul
+  - role: registrator
```

Comment:
i thought we were, as it is a docker container and registrator needs to register it in consul